src/PDFA-3-0/Cert/XIS-Server/medmij-pdfa-xis-2-2-serve-2-documentmanifest.xml

```diff
@@ -45,13 +45,13 @@
                 <value value="urn:oid:1.2.276.0.7230010.3.1.2.1787205428.8024.1526514975.220899"/>
             </assert>
         </action>
-        <action>
+        <action nts:in-targets="#default">
             <assert>
                 <description value="Confirm that the returned DocumentReference does not have a similiar masterIdentifier as the corresponding fixture."/>
                 <direction value="response"/>
                 <expression value="Bundle.entry.select(resource as DocumentManifest).where(type.coding.where(code ='73575-3')).masterIdentifier.value"/>
                 <operator value="notEquals"/>
-                <value value="urn:oid:1.2.276.0.7230010.3.1.2.1787205428.3024.1522314975.220879"/>
+                <value value="urn:oid:1.2.276.0.7230010.3.1.2.9857205428.8024.1526514975.440899"/>
```

Comment:
?

Reply:
Stond nog een verouderd ID in. Kan m.i. mee met deze wijziging.